### PR TITLE
Update youtube.mdx

### PR DESCRIPTION
**Pull request by fandyaditya**
Update documentation, now support get channel info and youtube info

## Description

Update documentation, now support getting Channel info and Video info

## Type of change

- [x] 📄 Update documentation

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [ ] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have appropriately commented on my code, especially in complex areas.
- [ ] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb/blob/staging/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Review by ZoranPandovski — APPROVED** [review]

File: docs/integrations/app-integrations/youtube.mdx [hunk 1]
```diff
@@ -53,3 +53,18 @@ WHERE youtube_video_id = "raWFGQ20OfA"
...
+
+
+Get information about any youtube video using video_id:
+```sql
+SELECT * FROM mindsdb_youtube.videos
+WHERE video_id="id";
+```
+
+
+Given a channel_id, get information about the channel
```
Comment (ZoranPandovski) [inline docs/integrations/app-integrations/youtube.mdx:65]: Please just add : at the end.

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]
Thanks. One minor change before we merge. Please also sign the CLA

File: docs/integrations/app-integrations/youtube.mdx (hunk 1 above)
Comment (fandyaditya) [reply docs/integrations/app-integrations/youtube.mdx]: all done!

**Comment by fandyaditya** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**